mods/d2k/sequences/structures.yaml

```diff
@@ -35,9 +35,9 @@ guntower:
 		Facings: -32
 		Offset: -24,16
 	muzzle: DATA.R8
-		Frames: 3839, 3839, 3840, 3840, 3841, 3841, 3842, 3842, 3843, 3843, 3844, 3844, 3845, 3845, 3846, 3846, 3847, 3847, 3848, 3848, 3849, 3849, 3850, 3850, 3851, 3851, 3852, 3852, 3853, 3853, 3854, 3854, 3855, 3855, 3856, 3856, 3857, 3857, 3858, 3858, 3859, 3859, 3860, 3860, 3861, 3861, 3862, 3862, 3863, 3863, 3864, 3864, 3865, 3865, 3866, 3866, 3867, 3867, 3868, 3868, 3869, 3869, 3870, 3870
+		Start: 3839
+		Tick: 60
 		Facings: -32
-		Length: 2
```

Comment:
You're right, on a second look the initial issue had with the cannon muzzle was that the Offset didn't fit.

> Use the muzzle sequence Offset: x,y to reset the center of the sprite to the center of rotation for the muzzle and turret (using no firing offset). Once this is set correctly then the muzzle should be consistently wrong for all facings.

The cannon muzzle anim has a "forward" value baked into the art (or .r8 offset) though, unless I fix the offset for each of the 32 facings individually (I don't really feel like doing that atm), either the bullet exit or muzzle position won't be perfect, the trike muzzle art is better in that regard. Just saying.